scripts/zones/Attohwa_Chasm/mobs/Lioumere.lua

```diff
@@ -2,13 +2,134 @@
 -- Area: Attohwa Chasm
 --  Mob: Lioumere
 -----------------------------------
-mixins = { require('scripts/mixins/families/antlion_ambush') }
------------------------------------
 ---@type TMobEntity
 local entity = {}
 
+local lioumereHome = { x = 478.8, y = 20, z = 41.7 }
+
+local pitAmbushSkillID = 278
```

Comment:
This is unneeded. We have an enum for it
`xi.mobSkill.PIT_AMBUSH_1`

Reply:
> This is unneeded. We have an enum for it `xi.mobSkill.PIT_AMBUSH_1`

Thanks for info, I have fixed the PR!